sys/net/gnrc/link_layer/lorawan/gnrc_lorawan_region.c

```diff
@@ -96,6 +96,10 @@ uint32_t gnrc_lorawan_pick_channel(gnrc_lorawan_t *mac)
 
 void gnrc_lorawan_process_cflist(gnrc_lorawan_t *mac, uint8_t *cflist)
 {
+#ifdef GNRC_LORAWAN_SINGLE_CHANNEL
```

Comment:
have you checked if the Join Procedure sends the channel list?
If so, there's no need add this guard.

Reply:
Yep, tested with TTN. The _Join_Accecpt_ contains the optional channel list.

Reply:
ah true, it's the same network server :)